topological_navigation_msgs/srv/UpdateEdgeConfig.srv

```diff
@@ -1,5 +1,5 @@
 string edge_id
-string namespace
+string namespace_name
```

Comment:
This is an annoying thing… but I see why it’s needed. Great you spotted that problem. It may require us to go through all our code searching for problems stemming from this change.